### DISCUSSION TITLE
Captcha refresh - should the refresh icons actually refresh captcha ?

### DISCUSSION THREAD

**Discussion by swjcarter** [ltb-project/self-service-password#1012] [Q&A]
When using the InternalCaptcha captcha.  Should the refresh circle actually refresh the captcha for when it is hard to read ?  When you inspect the circle it shows : <i id="captcha-refresh" class="fa fa-fw fa-refresh"></i>  But the circle is not clickable.

**Comment by davidcoutadeur** [comment]
That's the goal of the refresh arrows to refresh the captcha when it is hard to read.
It should be clickable.

Do you have a screenshot for showing how it looks?
Maybe there are errors in the javascript console that prevents the refresh?
